src/Linux/Avalonia.LinuxFramebuffer/Output/DrmBindings.cs

```diff
@@ -149,8 +150,24 @@ public DrmCard(string path = null)
         }
 
         public DrmResources GetResources() => new DrmResources(Fd);
+
+        /// <summary>
+        /// Pointer to the Graphics Buffer Management (GBM) Device for the DrmCard.  
+        /// </summary>
+        internal IntPtr GbmDevice
```

Comment:
GBM device handle should be either created in constructor or passed to constructor. In this case it should probably be a separate class with its own resource management.

Reply:
This has now been wrapped in a GbmDevice class. I have still had to expose a handle property though. This can only really be removed if the GbmDevice is responsible for creating the Egl Platform Interface instance from it.